### ISSUE TITLE
Redux required

### ISSUE THREAD

**Issue by SteffenBrinckmann** [jschr/electron-react-redux-boilerplate#90]
Hello,
I see that redux is used for all electron-react boilerplates. I like your boilerplate a lot since it is very small, so I wonder... can I pull out redux and use react-flux and the conventional routes? That would make the boilerplate even smaller and easier to understand (no container-components siblings required).
Is such approach possible or does electron require redux?
Thanks for the awesome work,
Steffen

**Comment by jschr** [comment]
Redux is not required to use electron. This boilerplate was created to show how to setup react and redux in electron but you can use whatever state management lib you prefer.

**Comment by roscoevanderboom** [comment]
@SteffenBrinckmann You can take a look at my fork of this boilerplate. I removed Redux, and I'm simply using react-router and react's useContext hook to manage state. It's not suitable for large application but works for small ones. I also added a tray window with ipc messaging from tray to mainWindow.  

https://github.com/roscoevanderboom/Electron-React-MaterialUI-Boilerplate

**Comment by SteffenBrinckmann** [comment]
@roscoevanderboom Thank you very much for the suggestion. I checked it out and it looks cool (due to MaterialUI) and it works great. Great job.
My goal is the other direction, though: simplify. no containers (just components), no tray-icon. MS Word and MS Code don't need it and they are very big applications. 
I found that simple programs are easier to debug and keep bug-free.
Thanks for the suggestion, though.
